docker-compose.yaml

```diff
@@ -1,25 +1,23 @@
 version: "3.9"
 
 networks:
-  ren-network: 
+  ren-network:
     name: ren-network
     driver: bridge
 
 services:
   ren:
-    container_name: ren
-    image: ren
-    restart: always
+    build:
+      context: .
+    restart: unless-stopped
     networks:
       - ren-network
     volumes:
-      - ren-data:/data
```

Comment:
I wouwd wike tuwu keep the dockew managed vowumes wathew than binding iwt diwectwy tuwu the host fiwesystem since messy stuff may potentiawwy occuw.
 
they pwobabwy won't, but iwt's genewawwy safew tuwu isowate the containew

Reply:
If we need to look into the actual folders I believe we do `docker volume ls` to list, then `docker volume inspect <volume_name>` to see where docker stored the information

Then to access it, we would need sudo privileges

Reply:
Reverted for the PR, but we might have to look at using bind mounts later because the plan is to mount the directory that contains `core/` and `cog/` without having to move the data.